build/hostScripts/dappnode_access_credentials.sh

```diff
@@ -0,0 +1,94 @@
+DAPPNODE_WELCOME_URL="http://welcome.dappnode"
+
+#############
+#1.FUNCTIONS#
+#############
+
+function dappnode_startup_delay () {
+  echo "Wait until DAppNode initializes..."
+  sleep 10
+}
+
+# $1 Connection method $2 Credentials
+function create_connection_message () {
+  echo -e "\n\e[32mConnect to DAppNode through $1 using the following credentials:\e[0m\n$2\n\nVisit \e[4m$DAPPNODE_ADMINUI_URL\e\n\n[0mCheck out all the access methods available to connect to your DAppNode at \e[4m$DAPPNODE_WELCOME_URL\e[0m\n"
+}
+
+function wifi_connection () {
+  # NOTE: network interface may be in use by host => $(cat /sys/class/net/${INTERFACE}/operstate)" !== "up")
+  # NOTE: wifi has a delay up to 1 min
+  # wifi container running
+  [ "$(docker inspect -f '{{.State.Running}}' ${WIFI_CONTAINER} 2> /dev/null)" = "true" ] && \
+  # Check interface variable is set
+  [ ! -z $(docker exec -it $WIFI_CONTAINER iw dev | grep 'Interface' | awk 'NR==1{print $2}') ] && \
+  create_connection_message "Wi-Fi" "$WIFI_GET_CREDS" && \
+  exit 0 || echo "Wifi not available"
+}
+
+function avahi_connection () {
+  # Ping to avahi endpoint: -c: number of pings. -w: timeout
+  avahi-resolve -n $AVAHI_ENDPOINT > /dev/null 2>&1 || { echo "Avahi-daemon not available" ; return ; }
+  # Https container exists
+  [ "$(docker ps -a | grep ${HTTPS_CONTAINER})" ] && \
+  # Https container running
+  [ "$(docker inspect -f '{{.State.Running}}' ${HTTPS_CONTAINER})" = "true" ] && \
+  # Https env variable LOCAL_PROXYING="true"
+  [ "$(docker exec -i ${HTTPS_CONTAINER} sh -c 'echo "$LOCAL_PROXYING"')" = "true" ] && \
+  # avahi-daemon running => systemctl is-active avahi-daemon RETURNS "active" or "inactive"
+  [ "$(systemctl is-active avahi-daemon)" = "active" ] && \
+  echo -e "\n\e[32mConnect to DAppNode through avahi-daemon.\e[0m\n\nVisit \e[4m$DAPPNODE_ADMINUI_LOCAL_URL\e\n\n[0mCheck out all the access methods available to connect to your DAppNode at \e[4m$DAPPNODE_WELCOME_URL\e[0m\n" && \
+  exit 0 || echo "Avahi-daemon not available"
+}
+
+function wireguard_connection () {
+  # wireguard container exists
+  [ "$(docker ps -a | grep ${WIREGUARD_CONTAINER})" ] && \
+  # wireguard container running
+  [ "$(docker inspect -f '{{.State.Running}}' ${WIREGUARD_CONTAINER})" = "true" ] && \
+  create_connection_message "Wireguard" "$($WIREGUARD_GET_CREDS)" && \
+  exit 0 || echo "Wireguard not available"
+}
+
+function openvpn_connection () {
+  # openvpn container exists
+  [ "$(docker ps -a | grep ${OPENVPN_CONTAINER})" ] && \
+  # openvpn container running
+  [ "$(docker inspect -f '{{.State.Running}}' ${OPENVPN_CONTAINER})" = "true" ] && \
+  create_connection_message "Open-VPN" "$($OPENVPN_GET_CREDS)" && \
+  exit 0 || echo "Open-VPN not available"
+}
+
+########
+#2.MAIN#
+########
+
+dappnode_startup_delay
+wifi_connection
+avahi_connection
+wireguard_connection
+openvpn_connection
+
+echo -e "\e[33mWARNING: no connection services available\e[0m"
```

Comment:
Maybe we should have a fallback here, like just show all of them or pick one. Showing this message is dangerous since the user won't have any action to take.

Reply:
You right, anyway this PR deppends on https://github.com/dappnode/DAppNode_Installer/pull/160

Once the installer PR is  merged and the DAppNode/release updated with the new assets this PR should change the `dappnode_access_credentials.sh`